src/test/java/com/acwong/peelplaylistcheck/playlist/PlayListHelperImplTest.java

```diff
@@ -0,0 +1,133 @@
+package com.acwong.peelplaylistcheck.playlist;
+
+import com.acwong.peelplaylistcheck.id.ContentId;
+import com.acwong.peelplaylistcheck.id.VideoTag;
+import com.acwong.peelplaylistcheck.model.Country;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PlayListHelperImplTest {
+    private static final String DEFAULT_TEST_VIDEO_CONTENT_INFO_FILE = "videolibrary1.json";
+
+    private PlayListHelperImpl helper;
+
+    @BeforeEach
+    public void setup() {
+        helper = new PlayListHelperImpl();
+    }
+
+    private File getContentCollectionFile(String fileFullPath) {
+        ClassLoader classLoader = getClass().getClassLoader();
+        return new File(classLoader.getResource(fileFullPath).getFile());
+    }
+
+    @Test
+    void loadDefaultVideoContentInfo() {
+        assertFalse(helper.getContentCollection() != null);
```

Comment:
assertNotNull is better?

Reply:
Yes.  Should better use assertNotNull.  Will update.